pkg/konfluxgen/pipeline-run.template.yaml

```diff
@@ -46,11 +46,10 @@ spec:
       value: 5d
     - name: output-image
       value: quay.io/redhat-user-workloads/ocp-serverless-tenant/{{{ truncate ( sanitize .ApplicationName ) }}}/{{{ truncate ( sanitize .ProjectDirectoryImageBuildStepConfiguration.To ) }}}:on-pr-{{revision}}
-    {{{- if ne .Pipeline "fbc-builder" }}}
     - name: build-platforms
       value:
         - linux/x86_64
-    {{{- else }}}
+    {{{- if eq .Pipeline "fbc-builder" }}}
     - name: skip-fbc-related-image-check
       value: "true"
```

Comment:
This should be removed as well as mentioned in https://github.com/openshift-knative/hack/pull/465/files#r1873091151